content/_index.md

```diff
@@ -15,7 +15,14 @@ container: "container-fluid"
 header_wrapper_class: "featured-jumbotron-home"
 ---
 {{< home/powered-by >}}
-{{< home/highlights >}}
+
+{{< home/highlights
+    id="featured-story-container"
+    publishTarget="ecd_tools"
+    templateId="featured-story-custom" 
+    count="5"
+    templatePath="/js/templates/featured-story-custom.mustache" >}}
```

Comment:
I would try to stay consistent with your naming conversation. You are calling this home/highlights but the template is featured-story-custom.

Reply:
I agree we need to be consistent. I'll rename the template based on what the shortcode will be named.